dockerfiles/nginx-tls/nginx.conf.tpl

```diff
@@ -29,7 +29,10 @@ http {
     ssl_ciphers         ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA384;
     ssl_certificate     /tmp/tls/cert.pem;
     ssl_certificate_key /tmp/tls/key.pem;
-    ssl_session_cache   shared:SSL:1m;
+    ssl_session_cache   shared:SSL:2m;
+
+    client_max_body_size    5m;
```

Comment:
the default is `1m`.  If the client sends a body bigger than `client_max_body_size`, nginx returns a `413` status code.  I strongly suspect that no normal client would ever need to exceed this but we can check if we've been returning many 413s.  If we don't need it I'm inclined to leave it at the default, because we don't want to allow malicious clients to send bigger requests than necessary.

Reply:
The nginx config used in old Verify infrastructure uses 2m

Reply:
We spoke in person and we think we can leave it at 1m. Happy for this to be implicit or explicit.